test/functional/web/safari-alerts-e2e-specs.js

```diff
@@ -34,45 +34,39 @@ describe('safari - alerts', function () {
   }
 
   it('should accept alert', async function () {
-    let el = await driver.elementById('alert1');
-    await el.click();
+    await driver.elementById('alert1').click();
```

Comment:
does this work?

Reply:
yes. All of this has passed in my fork of the repo (https://travis-ci.org/imurchie/appium-xcuitest-driver/builds/655921767)